app/lib/index.js

```diff
@@ -173,6 +167,41 @@ renderer.on('dialog:content-changed', function(done) {
   dialog.showDialog('contentChanged', done);
 });
 
+renderer.on('dialog:open-files', async function(options, done) {
+  const {
+    activeFile
+  } = options;
+
+  if (activeFile && activeFile.path) {
+    assign(options, {
+      defaultPath: path.dirname(activeFile.path)
+    });
+  }
+
+  const filePaths = await dialog.showOpenDialog(options);
+
+  done(null, filePaths);
+});
+
+renderer.on('dialog:save-file', async function(options, done) {
+  const { file } = options;
+
+  const { path } = file;
```

Comment:
This assignment shadows the `path` module. Easy fix for that is to use something like:

```javascript
const { path: filePath } = file;

if (filePath) {
  ...
```